speech/google/cloud/speech/result.py

```diff
@@ -46,9 +93,28 @@ def from_pb(cls, response):
         :rtype: :class:`~google.cloud.speech.result.StreamingSpeechResult`
         :returns: Instance of ``StreamingSpeechResult``.
         """
-        alternatives = [Alternative.from_pb(alternative)
-                        for alternative in response.alternatives]
+        alternatives = []
```

Comment:
Ah, I see here you changed it away from a list comprehension. Again just curious why.
Also can we change `/alt/alternative/`?